homeassistant/components/tado/sensor.py

```diff
@@ -124,6 +125,17 @@ def get_geofencing_mode(data) -> str:
         attributes_fn=lambda data: {"time": data["weatherState"]["timestamp"]},
         data_category=SENSOR_DATA_CATEGORY_WEATHER,
     ),
+    TadoSensorEntityDescription(
+        key="air comfort",
+        translation_key="freshness",
+        state_fn=lambda data: data["freshness"]["value"],
+        attributes_fn=lambda data: {
+            "lastOpenWidow": data["freshness"]["lastOpenWindow"],
+            "acPoweredOn": data["freshness"]["acPoweredOn"],
+            "lastAcPowerOff": data["freshness"]["lastAcPowerOff"],
```

Comment:
Can we maybe make separate sensors out of these attributes? We try to avoid putting too much in the attributes

Reply:
sure thing